<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompaniesActQR</class>
 <widget class="QWidget" name="CompaniesActQR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>464</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Companies Act</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icon/logo.png</normaloff>../Icon/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>711</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(250, 250, 250);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="sectiontag">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="text">
           <string>Section</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SectionInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="subsectiotag">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="text">
           <string>Sub Section</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SubSectionInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="clausetag">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="text">
           <string>Clause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ClauseInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="provisotag">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="text">
           <string>Proviso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ProvisoInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="explanationtag">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="text">
           <string>Explanation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ExplanationInput">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QFrame" name="ButtonGroup">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="UpdateDb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255,179,0);
color: rgb(255, 255, 255);
border-radius:10;</string>
        </property>
        <property name="text">
         <string>Update Database</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="Search">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color:rgb(0,137,123);
border-radius:10;</string>
        </property>
        <property name="text">
         <string>Search Companies Act</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../Resources/Images/searchbaar.png</normaloff>../../../Resources/Images/searchbaar.png</iconset>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="QuickSelection" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(254, 254, 254);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="QuickSelectPanel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(254, 254, 254);</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="RB_Quick">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>10</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quick but not maintained. Use this option only if you are sure that the particular section has not been amended&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Quick Reference</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="RB_Updated">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>10</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Slow but updated&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Updated Secton</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTextBrowser" name="SectionOutpit">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>11</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="TitleOutput">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>11</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Section Title</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>14</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(3,155,229);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Companies Act, 2013 - Quick Refer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTextBrowser" name="Footnote">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>11</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="overwriteMode">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SectionInput</tabstop>
  <tabstop>SubSectionInput</tabstop>
  <tabstop>ClauseInput</tabstop>
  <tabstop>ProvisoInput</tabstop>
  <tabstop>ExplanationInput</tabstop>
  <tabstop>RB_Updated</tabstop>
  <tabstop>RB_Quick</tabstop>
  <tabstop>TitleOutput</tabstop>
  <tabstop>SectionOutpit</tabstop>
  <tabstop>Footnote</tabstop>
  <tabstop>UpdateDb</tabstop>
  <tabstop>Search</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
